main.py

```
@@ -10478,3 +10478,66 @@ def __init__(self, *children, **kwargs):
+        reaction = CAT_REACTIONS[clean_item]
+    clean_item = item.strip()
+    safe_item = discord.utils.escape_markdown(
+        discord.utils.escape_mentions(clean_item)
+    )
+
+    if clean_item in CAT_REACTIONS:
+        reaction = CAT_REACTIONS[clean_item]
+        response_text = f"**You offered:** {safe_item}\n**Cat's Reaction:** {reaction['emoji']} {reaction['text']}"
+    else:
+        # Pick a random "not interested" response for any other emoji
+        random_response = random.choice(UNINTERESTED_RESPONSES)
```

Ruff (0.16.0)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)